contrib/epson-inkjet-printer-escpr/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "epson-inkjet-printer-escpr"
+pkgver = "1.8.2"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--disable-static",
+    "--with-cupsppddir=/usr/share/ppd",
+    "--with-cupsfilterdir=/usr/lib/cups/filter",
+]
+configure_gen = []
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = ["gmake"]
+makedepends = ["cups-devel"]
+depends = ["cups-filters"]
+pkgdesc = "Epson Inkjet Printer Driver (ESC/P-R)"
+maintainer = "triallax <triallax@tutanota.com>"
+license = "GPL-2.0-or-later"
+url = "https://download.ebz.epson.net/dsc/search/01/search?OSC=LX&productName=B700"
+source = f"https://download3.ebz.epson.net/dsc/f/03/00/15/37/36/9209d0195bbe0743c8f9f427bdc5d09186d72ba2/epson-inkjet-printer-escpr-{pkgver}-1.tar.gz"
```

Comment:
is this link stable

Reply:
Unfortunately not, it changes on every version. :/

Reply:
changing every version doesn't matter and wasn't the question, i'm asking if this link will remain the same for this version and if it'll provide a tarball of the same checksum

Reply:
Ah, if that's what what you mean, then yes, the link is stable.